templates/includes/delete_confirm.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+Delete
+{% block content %}

```

Comment:
Consider adding a message or context within the `content` block to inform the user about what specific item is being deleted. This can help prevent accidental deletions by providing more clarity.